internal/cli/login.go

```diff
@@ -63,6 +63,14 @@ func RunLogin(ctx context.Context, cli *cli, expired bool) error {
 	cli.renderer.Infof("Successfully logged in.")
 	cli.renderer.Infof("Tenant: %s\n", res.Tenant)
 
+	// store the refresh token
+	secretsStore := &auth.Keyring{}
+	err = secretsStore.Set(auth.SecretsNamespace, res.Tenant, res.RefreshToken)
+	if err != nil {
+		// log the error but move on
+		cli.renderer.Warnf("Could not store the refresh token locally, please expect to login again once your access token expired: %s.", err.Error())
+	}
+
```

Comment:
Isn't it necessary to set the `secretsStore` instance on `a.Secrets`?

Reply:
not anymore, because the token is stored outside of the _authorizer_ since an error is not interrupting the whole flow.

but thanks for asking this, I missed a cleanup there in the struct definition